test/mocha/14-keystores-getStorageUsage-api.js

```diff
@@ -56,5 +56,34 @@ describe('keystores APIs', () => {
       result.should.be.an('object');
       result.should.deep.equal({storage: 2});
     });
+    it('gets custom storage usage for a keystore when max concurrency is ' +
+      'reached', async () => {
+      let err;
+      let result;
+      for(let i = 3; i < 54; i++) {
```

Comment:
Why `54`? That your favorite number? :)

It seems like this test can't be run independently ... is that right? Perhaps we should clear storage before running this test?

Reply:
It can be run independently and 54 is used here to a counters.length of 102 is used here to make sure the max concurrency is reached. I started at 3 because the other tests are using 1 and 2, so there is no duplicate issue via `id`. So using 54 here allows the while loop to be run 51 times and since aggregate is true, it will create a counter.length of 102. 54 is not arbitrary but I can add a comment about this.

Reply:
+1, the comment clears it up. Also, if the value of `100` ever changes then this test will need to change.